docs/Migrating to Newer SDL Versions/Migrating SDL Core 7.0 to 7.1/index.md

```diff
@@ -0,0 +1,231 @@
+        Either menuParams or vrCommands must be provided.
+    </description>
+    
+    <!-- New Parameters -->
+    <param name="secondaryImage" type="Image" mandatory="false">
+        <description>Optional secondary image struct for menu cell</description>
+    </param>
+</function>
+
+<struct name="MenuParams" since="1.0">
+    <!-- New Parameters -->
+    <param name="secondaryText" maxlength="500" type="String" mandatory="false">
+        <description>Optional secondary text to display</description>
+    </param>
+    <param name="tertiaryText" maxlength="500" type="String" mandatory="false">
+        <description>Optional tertiary text to display</description>
+    </param>
+</struct>
+```
+
+#### Broadening choice uniqueness
+
+Prior to SDL Core 7.1, choice set choices and menu commands were required to have unique primary text. SDL Core 7.1 removes this restirction 
+
+#### Keyboard Enhancements
+
+SDL Core 7.1 adds a new `NUMERIC` keyboard layout and new enhancements to allow apps to mask entered characters and change special characters shown on the keyboard layout.
+
+### OEM exlcusive apps support
+
+SDL Core 7.1 adds the ability to share vehicle type information before sending the Register App interface request. This will enable SDL adopters to provide exclusive apps to their users depending on vehicle type
+
+The vehicle type information parameters have been added to the BSON payload of the `StartServiceACK` protocol message
+
+|Tag Name|Type|Description|
+|--------|----|-----------|
+|make|String|Vehicle make|
+|model|String|Vehicle model|
+|model year|String|Vehicle model year|
+|trim|String|Vehicle trim|
+|systemSoftwareVersion|String|Vehicle system software version|
+|systemHardwareVersion|String|Vehicle system hardware version|
+
+The vehicle type information parameters (`vehicleType` and `systemSoftwareVersion`) in `RegisterAppInterface` have been deprecated
+### Video streaming capability updates
+
+#### Preffered FPS
+
+- Added new parameter `prefferedFPS` to the `VideoStreamingCapability` struct.
+
+```xml
+<struct name="VideoStreamingCapability" since="4.5">
+    <description>Contains information about this system's video streaming capabilities.</description>
+    ...
+    <!-- new param -->
+    <param name="preferredFPS" type="Integer" minvalue="0" maxvalue="2147483647" mandatory="false">
+        <description>The preferred frame rate per second of the head unit. The mobile application / app library may take other factors into account that constrain the frame rate lower than this value, but it should not perform streaming at a higher frame rate than this value.</description>
+    </param>
+</struct>
+```
+
+#### Updating video streaming capabilites during ignition cycle
+
+SDL Core 7.1 adds the ability to update video streaming capabilities during the ignition cycle. This will allow uses cases which require dynamic resolution switching (Picture-in-Picture, preview, split screen, etc.)
```

Comment:
```suggestion
SDL Core 7.1 adds the ability for an application to update its video streaming capabilities during the ignition cycle. This will allow SDL to handle uses cases which require dynamic resolution switching (Picture-in-Picture, preview, split screen, etc.)
```